src/DynamoCore/Extensions/ReadyParams.cs

```diff
@@ -76,6 +77,17 @@ public virtual ICommandExecutive CommandExecutive
             get { return commandExecutive ?? (commandExecutive = new ExtensionCommandExecutive(dynamoModel)); }
         }
 
+        /// <summary>
+        /// Returns Sessions Linter Manager
+        /// </summary>
+        public LinterManager LinterManager
```

Comment:
so this has a change in behavior, not just where data is stored - ready is only called once per Dynamo session lifetime.

Reply:
If you really want this in StartupParams why not just add a new constructor that takes a bag of options or other objects.

Reply:
Let's do that...  We can add it to the StartupParams with all the other manager components